cmd/users.go

```diff
@@ -22,9 +22,12 @@ func (d DeisCmd) UsersList(results int) error {
 		return err
 	}
 
-	d.Printf("=== Users%s", limitCount(len(users), count))
+	d.Printf("=== Users (* admin)%s", limitCount(len(users), count))
```

Comment:
Yeah, I'm just saying that it didn't make sense to me on first glance. Maybe `(* == admin)`?